GP24-102-CSharp-Basic/NimC/Program.cs

```diff
@@ -0,0 +1,426 @@
+﻿char menuNavigation = ' ';
+bool isGameRunning = true;
+bool isPlayerGreenTurn = true;
+bool bothPlacedShips = false;
+int[] originalShipTypes = { 5, 4, 3, 2, 2};
+int[] usedShipTypes;
+
+string letterRange = "abcdefghij";
+
+string[,] gridGreen = 
+{
```

Comment:
Is this actually a working version of Battleships? That's mindblowing! I'm at a loss for words!